ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/svi_profiles.schema.yml

```diff
@@ -0,0 +1,33 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  svi_profiles:
+    documentation_options:
+      filename: Network Services
+    $ref: "#/$defs/svis"
```

Comment:
I think it would be better to have a `defs` with the singluar `svi` item instead. That way we would not have to inherit the whole list definition, but just have the `$ref` on the `items` key.